app/views/layouts/application.html.erb

```diff
@@ -8,7 +8,7 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'showTime': 'reload' %>
+    <%#= javascript_pack_tag 'application', 'showTime': 'reload' %>
```

Comment:
If you don't occupy this line, delete it.

Reply:
This line is need it.